pkg/kubeconfig/create_kubecfg_test.go

```diff
@@ -330,20 +334,12 @@ func TestBuildKubecfg(t *testing.T) {
 			name: "Test Kube Config Data For Gossip cluster with admin and secondary NLB port",
```

Comment:
@olemarkus Does this test make sense to you? Gossip implies cluster without DNS. You would not have a public name with Gossip and adding a cert would be pretty hard.